daemon/api.go

```diff
@@ -281,6 +281,11 @@ func getSnapsInfo(c *Command, r *http.Request) Response {
 		includeLocal = true
 	}
 
+	var includeTypes []string
+	if len(query["types"]) > 0 {
```

Comment:
I think the comma-separated variant is maybe a bit more intuitive. As long as we're consistent we're good though!